### PR TITLE
pushing the updated code for booking and package_availability functions

### PR DESCRIPTION
**Pull request by Spandukm**
Creating a pull request for merging the code regarding the booking and package_availability functions to development branch.

### CODE REVIEW

File: include/booking.h [hunk 1]
```diff
@@ -1,6 +1,54 @@
...
+ */
+ int booking(void);
 
 /**
-* @brief Booking of the requested number of seats with requested type of rooms and also generates the unique booking ID for all the confirmed booking
+*   File name
 */
-void booking();
\ No newline at end of file
+ #define filename "Details.csv"
```
Comment (thej-rayasam) [inline include/booking.h:27]: Can you please add the csv file into the repository

File: src/booking.c [hunk 2]
```diff
@@ -1,47 +1,224 @@
...
+                                        printf("Requested number of seats are available!\n");
+                                        booking_id = passenger_details(no_of_pass);
+                                        /*
+                                        * To Do -- add funtions suite_selection and fare_calcultion
+                                        */
+                                        break;
+                                    }else{
+                                         printf("Sorry:( %d seats are left\n",  available);
+                                         break;
+                                    }
```
Comment (thej-rayasam) [inline src/booking.c]: There is a continue condition in the flowchart which is missing in the code. Can you please update it.

File: include/booking.h (hunk 1 above)
Comment (Spandukm) [reply include/booking.h:27]: Ok. Added the file into the repository.

File: src/booking.c (hunk 2 above)
Comment (Spandukm) [reply src/booking.c]: Updated file with missing condition.

**Review by thej-rayasam — APPROVED** [review]

**Comment by swethameenakshinatarajan** [comment]
The cruise option entered by the user should be verified to be valid.
The package availability function seems to be taking the number of seats of the next cruise into consideration.

**Pull request merged**